### ISSUE TITLE
Custom Date filter not working as expected

### ISSUE THREAD

**Issue by Shubhdeep12** [getsentry/sentry#59814]
### Environment

SaaS (https://sentry.io/)

### Steps to Reproduce

Open the custom date filter but do not apply - it gets applied
Click back but then also the filter is getting applied.

### Expected Result

Filter should be applied when clicked in Apply button in case of Custom date filter.

### Actual Result

https://vimeo.com/883870250/78573c63d3?share=copy

### Product Area

Issues

**Comment by Shubhdeep12** [comment]
Please confirm if it is a valid bug. I would love to fix this.

**Comment by ceorourke** [comment]
Thanks for the report - I've reproduced it and put it in our backlog to be addressed.

**Comment by Shubhdeep12** [comment]
HI @ceorourke thanks for the confirmation.

is this repository open for contributions from non-sentry members?

**Comment by ceorourke** [comment]
@Shubhdeep12 yes we accept PRs from anyone - if it's not automatically done, you can assign the review to the @getsentry/issues team

**Comment by Shubhdeep12** [comment]
Hi @ceorourke Thanks for the confirmation.
Raised the PR. 
Seems like there is no option to assign a review to @getsentry-issues.

**Comment by cvxluo** [comment]
Going to close this since it is intentional behavior — see https://github.com/getsentry/sentry/pull/101090#pullrequestreview-3311278593

heads up @shayna-ch
